doc/01-Basic-Menus.markdown

```diff
@@ -310,16 +310,7 @@ $routeVoter->setRequest($symfonyRequest);
 Creating a Menu from a Tree structure
 -------------------------------------
 
-You can create a menu easily from a Tree structure (a nested set for example) by
-making it implement `Knp\Menu\NodeInterface`. You will then be able
-to create the menu easily (assuming `$node` is the root node of your structure):
-
-```php
-<?php
-
-$factory = new \Knp\Menu\MenuFactory();
-$menu = $factory->createFromNode($node);
-```
+See [Advanced Menu documentation page](01a-Advanced-Menu.md)
```

Comment:
This would force to change every subsequent number, that I guess it's not good (since it would break links coming from external source). I could even move it to last number (number 6), but I guess that it would make more sense to keep similar arguments next each others.